packages/angular/projects/ui-angular/src/lib/services/authenticator.service.ts

```diff
@@ -155,26 +156,23 @@ export class AuthenticatorService implements OnDestroy {
 
   /** @deprecated For internal use only */
   public get slotContext() {
-    const slotContext = {
-      ...this._facade,
-      ...this._sendEventAliases,
-    };
-
     return {
-      ...slotContext,
-      $implicit: { ...slotContext },
+      ...this._facade,
+      $implicit: this._facade,
     };
   }
 
-  /** @deprecated For internal use only */
-  public subscribe(callback: AuthSubscriptionCallback) {
-    if (this._authService) {
-      return this._authService.subscribe(callback);
-    } else {
+  public subscribe(callback: AuthSubscriptionCallback): () => void {
+    if (!this._authService) {
       logger.error(
         'Subscription attempted before machine was created. This is likely a bug on the library, please consider filing a bug.'
       );
     }
+
+    const subscription = this._authService.subscribe(() => {
+      callback(this._facade);
+    });
+    return subscription.unsubscribe;
```

Comment:
Here's the actual feature; we return `_facade` instead of internal state machine state value.

Note that we return `unsubscribe` directly so customers can just do `const unsubscribe = this.authenticator.subscribe()`.